apps/meteor/client/views/omnichannel/directory/contacts/ContactTable.tsx

```diff
@@ -64,8 +63,13 @@ function ContactTable(): ReactElement {
 			}),
 	);
 
-	const { reload, ...result } = useEndpointData('/v1/livechat/visitors.search', { params: query });
-
+	const {
+		data: contactResult,
+		isLoading: isLoadingContacts,
+		isError: contanctTableError,
+		isSuccess,
+		refetch: contactReload,
```

Comment:
I'd ask you to change this variable names to follow the same pattern. If you want to be explicit you could use the names `isLoadingContacts`, `isErrorContacts`, `isSucessContacts`, `reloadContacts`. 

Also for this specific component I see no problem in keeping them simply as `isLoading`, `isError`, `isSuccess`, `refetch`, since it's the only fetch present.

Reply:
hi @aleksandernsilva thank you for reviewing I have updated the variable name as you suggested